website/docs/developer/deployment-guide/setting-up-deployment-environment.md

```diff
@@ -0,0 +1,72 @@
+---
+id: deployment-environment
+title: Deployment Environment
+description: Provides developer with information on setting up deployment environment.
+---
+
+In order to deploy ORCA into an NGAP environment with Cumulus, a deployment
+environment needs be created. The sections below describe the tool chain needed
+and any additional environmental setup that should occur. The setup for deploying
+ORCA is similar to the [Cumulus deployment requirements](https://nasa.github.io/cumulus/docs/deployment/deployment-readme#requirements).
+The sections below assume that the end user is using a Linux or MacOS environment.
+
+
+## Deployment Tool Requirements
+
+The following tools should be installed to perform an ORCA deployment.
+
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- zip/unzip
+- [AWS CLI](https://aws.amazon.com/cli/)
+- [AWS CLI Session Manager](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html)
+- [Terraform Version Manager](https://github.com/tfutils/tfenv)
+- [Terraform](https://www.terraform.io/intro/index.html) v0.13.6 and up.
```

Comment:
Since this step has extra instructions, should we note that and link down to the proper step?

Reply:
I will add some additional verbiage and a link.